clusterloader2/pkg/util/cluster.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+)
+
+// GetNodesNumber returns number of nodes in the cluster.
+func GetNodesNumber(c clientset.Interface) (int, error) {
+	nodeList, err := c.CoreV1().Nodes().List(metav1.ListOptions{})
+	if err != nil {
+		return 0, err
+	}
+	numNodes := 0
+	for i := range nodeList.Items {
+		if nodeList.Items[i].Spec.Unschedulable == false {
```

Comment:
There is question what about not-ready nodes (or in general, nodes that has taints preventing scheduling on that nodes).

In load tests (consciously) we decided for:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/load.go#L148

I would say - let's do the same.

Reply:
Done.
I copied the logic from test/framework. It created a lot of dependencies, though.